### ISSUE TITLE
FileAPI syntax error: api is not defined

### ISSUE THREAD

**Issue by gkaemmer** [danialfarid/ng-file-upload#1163]
Seems like there's a syntax error at https://github.com/danialfarid/ng-file-upload/blob/master/src/FileAPI.js#L278. Here's the code around line 278:
```
		...
		_supportInputAttr = function (attr){
			var input = document.createElement('input');
			input.setAttribute('type', "file");
			return attr in input;
		},

		_support // Why is this line here??

		/**
		 * FileAPI (core object)
		 */
		api = {
		...
```
Gives the error: `Uncaught ReferenceError: api is not defined`.

Is this working for other people? Seems like it could be fixed by just deleting line 278.

**Comment by danialfarid** [comment]
I think minified version gets rid of that.

**Comment by danialfarid** [comment]
Fixed at version 10.0.3
